hbase-to-mongo-export/htme-host-install.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh -x
+
+## Script to install htme service
+
+# Install Java
+sudo yum update -y
+sudo yum install -y java-1.8.0-openjdk-devel
+
+# Install Amazon SSM agent
+sudo yum install -y https://s3.amazonaws.com/ec2-downloads-windows/SSMAgent/latest/linux_amd64/amazon-ssm-agent.rpm
+
+# Install acm cert helper
+acm_cert_helper_repo=acm-pca-cert-generator
+acm_cert_helper_version=0.8.0
+
+# pip is not available in CentOS 7 core repositories there is a requirement to enable EPEL repositories prior
+sudo yum --enablerepo=extras install -y epel-release
+sudo yum install -y python-pip
+
+# gcc and python-devel are required to enable the twofish indirect dependency -
+# of acm-pca-cert-generator to be built and installed
+sudo yum install -y gcc
+sudo yum install -y python-devel
+sudo pip install https://github.com/dwp/${acm_cert_helper_repo}/releases/download/${acm_cert_helper_version}/acm_cert_helper-${acm_cert_helper_version}.tar.gz
+sudo yum remove -y gcc python-devel
+
+# Download & install AWS-CLI
```

Comment:
lines 27-34 can be replaced with 
`pip install awscli`

Reply:
ah cool ta, assumed there was some reason we'd not done that so just went with what's in DKS :)